### PR TITLE
Add a section on how we manage engineers

### PR DESCRIPTION
**Pull request by Reettaphant**
Adds a page on how we manage engineers. I tried to follow similar markdown to the engineering culture one so that the pages look similar but the bullet points don't work quite as well for a text with longer sentences. I'm also happy to trim down the sentences if we want to enforce similar style of writing across pages.

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
👍

File: src/engineering.ejs [hunk 1]
```diff
@@ -19,6 +19,7 @@
                             <ul class="base-list">
                                 <li class="trailing-margin">
                                     <h3 class="alt-h3"><a href="engineering-culture.html">Culture</a></h3>
+                                    <h3 class="alt-h3"><a href="managing-engineers.html">Culture</a></h3>
```
Comment (LATaylor-guardian) [inline src/engineering.ejs]: Link text can be changed to something more appropriate here.

File: src/content/managing-engineers.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+**What’s the difference between an Engineering Manager, a Product Manager and a Scrum Master?**
+- **Engineering Managers** are accountable for the engineering performance of software systems, for the technical decisions made in the team and for line managing engineers - the “technical how” if you like.
+
+- **Product Managers** are responsible for the high level direction of the product and facilitate relationships between engineers and stakeholders. They generally own the “what” of the team.
+
+- **Scrum Masters** facilitate the smooth running of the team, helping to unblock issues, take a lead on external comms and organise team meetings - we could characterise this as the “process how”.
+
+**Still Confused? —** Just have a word with your line manager who should be able to talk you through all of the processes and procedures we have around management, or at the very least will find out and get back to you.
```
Comment (LATaylor-guardian) [inline src/content/managing-engineers.md]: This part reads like we're talking to one of our reports as opposed to someone reading this who is a prospective Guardian applicant/joiner.

File: src/content/managing-engineers.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+**What sort of management should I expect?**
+- The way that engineers in teams work together is collaborative and open: we encourage people to voice their opinions and teams work together to find solutions to problems that everyone is happy with. The engineering manager in the team is ultimately accountable for the direction the team travels and makes sure that teams decisions are well considered and align with the goals of the organisation.
+
+- You and your line manager should schedule one to one meetings on a regular basis, typically fortnightly though the frequency will vary depending on how long you’ve been with the organisation or the team, and on if you have a  preference or a need for more frequent contact. These meetings are an opportunity for you to raise any concerns you may have, discuss your career progression, identify training and development needs and really anything else that is affecting your working life. It’s also an opportunity for your manager to give you feedback on your performance - things that are working well and also areas where you would benefit from a change.
```
Comment (davidfurey) [inline src/content/managing-engineers.md]: Maybe `should schedule` -> `will schedule`

File: src/content/managing-engineers.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- Requests for holidays, notification of sickness and any changes for the way you work should go through your line manager, but it’s also a great idea to keep your team informed to ensure that your plans won’t cause a problem. As an organisation we like to say yes to reasonable requests but we also need to balance these against the needs of the business which your manager acts as a proxy for, and we need to ensure that everyone is treated fairly when dealing with these requests.
```
Comment (davidfurey) [inline src/content/managing-engineers.md]: `for the way you work should go through` -> `for the way you work go through`

File: src/content/managing-engineers.md (hunk 2 above)
Comment (davidfurey) [reply src/content/managing-engineers.md]: We could just drop this paragraph, or replace it with something more generic about how to find out more about working at the Guardian.

Comment (Reettaphant) [reply src/content/managing-engineers.md]: I'm going to remove this section, I think this document is already quite long as it is.

**Comment by LATaylor-guardian** [comment]
👍

**Comment by franziskas** [comment]
Great work Reetta!

**Pull request merged**